.eslintrc

```diff
@@ -14,53 +18,35 @@
   },
   "rules": {
     // Override airbnb style.
-    "arrow-body-style": 0, // Leave bracing to code reviewer discretion.
     "camelcase": [2, { "properties": "always" }],
-    "comma-dangle": 0, // Dangling commas are wonderful. It's JSON that's stupid.
     "func-names": 2,
+    "import/no-extraneous-dependencies": 0,
     "no-console": 2,
     "no-negated-condition": 0, // Sometimes negated conditions are easier to understand.
+    "no-plusplus": 0,
+    "no-underscore-dangle": 0, // We have build flags that use this
     "no-unused-vars": [
       2,
       { "args": "after-used", "argsIgnorePattern": "^_", "vars": "local" }
     ],
     "prefer-rest-params": 2,
-    "quote-props": [2, "as-needed", { "keywords": true }],
-    "space-unary-ops": 2,
-    "max-len": [0], // No agreed upon max length and react lines get > 100 chars easily.
-    "arrow-parens": 0, // We don't have an agreed upon pattern here
-    "no-underscore-dangle": 0, // We have build flags that use this
+    "quote-props": [2, "as-needed", { "keywords": false }],
```

Comment:
What's the thinking behind changing this? I've encountered the keyword error a few times. Thanks for any insights.

Reply:
For some time now it's been totally valid to use reserved words as unquoted property names. So no need to put `enum` in quotes, for example. https://stackoverflow.com/questions/4298594/javascript-keywords-in-hash-keys